presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PruneIdentityProjections.java

```diff
@@ -48,6 +48,18 @@ public PlanNode optimize(PlanNode plan, Session session, Map<Symbol, Type> types
         return SimplePlanRewriter.rewriteWith(new Rewriter(), plan);
     }
 
+    public static boolean isIdentityProjection(ProjectNode node)
```

Comment:
I have moved it to `ProjectNode`. Having separate class for just this check is also weird.